documentation/contributing/formatting-guide.adoc

```diff
@@ -0,0 +1,39 @@
+[[formatted-text-reference]]
+= AsciiDoc Format Reference
+
+Tips for which AsciiDoc markup to use when formatting text.
+
+Monospace (```)::
+[cols="50%,50%",options="header"]
+|===
+|Item |Example
+|File names and paths |The \`/home/user/.bashrc` file ...
+|Literal values | If the value is \`true` ...
+|Configuration attributes | Set the \`enabled` attribute ...
+|Java class names |The \`java.sql.Connection` class ...
+|Java annotations |The \`@Clustered` annotation ...
```

Comment:
I assume we should also use monospace for Kubernetes/OpenShift resource kinds (`Pod`, `ConfigMap` etc). Can we add that as an example so it's completely clear to contributors? Please could you also open an issue to convert the existing docs to using monospace for resource kinds?

Reply:
Let's add 2 new issues - 1 for a better example and 1 for fixing the existing docs.  @dmuntima and I are going through the existing docs to help convert them to templates and make edits like this, but would love some help!!!

Reply:
I raised #609 and #608. Are GitHub issues enough for this? Or do we need JIRAs?